lib/tester.js

```diff
@@ -21,14 +21,24 @@ class Tester {
     }
     this.schema = schemaParser(schema)
     this.mockedSchema = mocker(schema)
+    this.fixture = null
+    this.opts = {}
   }
 
   mock (opts, args = {}) {
     if (!opts || (isObject(opts) && !opts.query && !opts.kind)) {
       throw new Error('The Query/Mutation to test is require')
     }
 
-    let { query, variables, fixture, saveFixture = false } = opts
+    const operationOptions = Object.assign({}, this.opts, opts)
```

Comment:
you can use spread here

{...this.opts, ...opts}

Reply:
Thanks!! 🎉